rollup.config.js

```diff
@@ -9,23 +9,11 @@ export default [{ // Vanilla JS source
     input: 'lib/js/app.js',
     output: [
         {
-            file: 'lib/js/dist/SDL.min.js',
+            file: 'dist/js/SDL.min.js',
```

Comment:
@renonick87 after replacing this path, all tests are broken. There is required to replace the imports of `SDL.min.js` in all occurrences. I've created an issue #300 and PR #301 with the necessary fix. Please review and merge.